simulator/mips/t/unit_test.cpp

```diff
@@ -9,6 +9,15 @@ TEST_CASE( "MIPS32_instr_init: Process_Wrong_Args_Of_Constr")
     CHECK_NOTHROW( MIPS32Instr(0x0).execute());
 }
 
+TEST_CASE( "sllv")
+{
+    MIPS32Instr instr("sllv");
+    instr.set_v_src( 1, 0);
+    instr.set_v_src( 0, 1);
+    instr.execute();
```

Comment:
Could you please add a test case for non-existing instruction?
It should throw an exception if `execute()` is called.